### DISCUSSION TITLE
How add a dash to the beginning of all lines of the text box except the first line in vb .net?

### DISCUSSION THREAD

**Discussion by VahidEra** [dotnet/wpf#7464] [Q&A] [answered]
hi guys,
i have a multiline textbox in my visualbasic project like this:
```
line1
line2
line3
line4
```

and i want to add a dash to beginig of all lines execpt the first line like this:

```
line1
- line2
- line3
- line4
```

with loop like ``` for each line in textbox1.lines``` add dash to all lines, but i want to add dash to all lines except line one.

**I have no idea how to do this**
thank you helping me guys.

**Comment by miloush** [comment, marked as answer]
Assuming you don't care about performance very much, you would need:
1. split the string using `String.Split`, use e.g. `Environment.NewLine` as a seperator, this will give you an array of strings
2. go through the array and add dashes at the beginning of the strings you want (i.e. replace the element in the array with a new string)
3. join the array back into single string using `String.Join` and `Environment.NewLine` as a separator
4. set the `TextBox.Text` to the joined string

**Reply by VahidEra** [reply]
@miloush Dear friend thank you very much.
that is exactly what i wanted.
